src/libraries/System.Private.Xml/src/System/Xml/Core/XmlUtf8RawTextWriter.cs

```diff
@@ -710,6 +710,23 @@ private void FlushEncoder()
 
         }
 
+        internal override unsafe void WriteSpan(ReadOnlySpan<char> value)
```

Comment:
Honestly I'd go into direction of getting rid of any pointer usage and this is adding one. Could we attempt to replace `WriteAttributeTextBlock` and `WriteElementTextBlock` to use span instead?